lib/Quantization/Serialization.cpp

```diff
@@ -35,11 +35,33 @@ template <> struct MappingTraits<glow::NodeQuantizationInfo> {
   }
 };
 
+template <> struct BlockScalarTraits<glow::MultiLineStr> {
```

Comment:
that does not belong to quantization/serialization. Could you move that to a separate configs, lib/configs

Reply:
also add a test for this.